### ISSUE TITLE
Plugin Error: Can't find `registerPlugin` when requiring other files that call registerPlugin

### ISSUE THREAD

**Issue by aaronfrost** [scullyio/scully#124]
# 🐞 Bug report
When you are in a separate file creating a plugin and you call `registerPlugin` to register the plugin, you get an error when you run `npm run scully`. 

### Description
In order to have a healthy plugin system, we need to enable people to have plugins in other own contained files. Some of their plugins will be __Self Registered Plugins__, meaning that the plugin will internally call `registerPlugin` to get it registered with Scully. 

Currently, when I do this, I get an error stating that it can't find the global `registerPlugin`, even though I can see that the globals have been added. 

## 🔬 Minimal Reproduction
Create another file, and in it add:
```javascript
registerPlugin('router', 'foo', () => Promise.resolve([]));
```
Then require this file in your `scully.config`. You will see the error. 

## 💻Your Environment 
**Angular Version:**
<pre><code>

Angular CLI: 9.0.0-rc.6
Node: 10.14.2
OS: darwin x64

Angular: 9.0.0-rc.6
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router
Ivy Workspace: Yes

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.900.0-rc.6
@angular-devkit/build-angular     0.900.0-rc.6
@angular-devkit/build-optimizer   0.900.0-rc.6
@angular-devkit/build-webpack     0.900.0-rc.6
@angular-devkit/core              9.0.0-rc.6
@angular-devkit/schematics        9.0.0-rc.6
@angular/cdk                      8.2.3
@ngtools/webpack                  9.0.0-rc.6
@schematics/angular               9.0.0-rc.6
@schematics/update                0.900.0-rc.6
rxjs                              6.5.3
typescript                        3.6.4
webpack                           4.41.2
</code></pre>

**Scully Version:**
<pre><code>

"@scullyio/scully": "0.0.47",
</code></pre>
## 🔥 Exception or Error
<pre><code>

</code></pre>
![image](https://user-images.githubusercontent.com/662832/71608097-b7817180-2b3b-11ea-86f1-5dd07259c628.png)

**Comment by aaronfrost** [comment]
I tried this with the changes from #107 and it didn't fix the issues.

**Comment by SanderElias** [comment]
would this be sufficient? 
in `scully config.js`:
```JS
require('./extraPlugin/extra-plugin.js')(registerPlugin);
```

in `./extraPlugin/extra-plugin.js'`
```JS
/** plugin code here **/
module.exports = registerPlugin => registerPlugin('router', 'extra', extraRoutesPlugin);
```

Works with #107

**Comment by SanderElias** [comment]
would this be sufficient? 
in `scully config.js`:
```JS
require('./extraPlugin/extra-plugin.js')(registerPlugin);
```

in `./extraPlugin/extra-plugin.js'`
```JS
/** plugin code here **/
module.exports = registerPlugin => registerPlugin('router', 'extra', extraRoutesPlugin);
```

Works with #107

**Comment by SanderElias** [comment]
closed in favor of  #122. Let us center discussion there
